### PR TITLE
Noto Sans Symbols 2: Version 2.005; ttfautohint (v1.8.4.7-5d5b) added

### PR DESCRIPTION
**Pull request by simoncozens**
72cb38d: [gftools-packager] Noto Sans Symbols2: Version 2.004; ttfautohint (v1.8.4.7-5d5b) added

* Noto Sans Symbols2 Version 2.004; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/notofonts/symbols.git at commit https://github.com/notofonts/symbols/commit/.

### CODE REVIEW

**Comment by simoncozens** [comment]
Ignore the dotted circle fail. This dotted circle is used as the dotted circle symbol, not the mark-holding placeholder. (It's significantly bigger, for example.)

File: ofl/notosanssymbols2/METADATA.pb [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
 subsets: "symbols"
-is_noto: true
-sample_text {
-  masthead_full: "♠🖫🙥👍"
-  masthead_partial: "🏆🡽"
-  styles: "🌍✄✎ 🏔🏕🏌🏍🎭🎮 🯅🯆🯇🯉 🡢🡭🡱🡼 🯱🯲🯳🯴🯵🯶 🂮🂱🂲🂳"
-  tester: "✷✺✻✽ ⯮⮊⮫⬈⇨🡲 ♠♣🖪🖫 🨀🨁🨂🨃🨄🨅 🗪🗮🗰 ⯞⯟⯠⬥⬦⬧ ⢈⢉⢊⢋⣋⣌⣍ ◻◼🞔🞕🞖 𐇠𐇡𐇢𐇣𐇤𐇗𐇘"
-  poster_sm: "🡢🡱🏠💻🐿👁📽"
-  poster_md: "⌚✋⯧☔🛪🏟⛅🞽🕖🚲"
-  poster_lg: "🡽🨄"
```
Comment (RosaWagner) [inline ofl/notosanssymbols2/METADATA.pb:25]: Is it okay to remove the sample text?

File: ofl/notosanssymbols2/METADATA.pb [hunk 2]
```diff
@@ -1,26 +1,23 @@
-name: "Noto Sans Symbols 2"
+name: "Noto Sans Symbols2"
 designer: "Google"
 license: "OFL"
 category: "SANS_SERIF"
 date_added: "2020-11-19"
 fonts {
-  name: "Noto Sans Symbols 2"
+  name: "Noto Sans Symbols2"
```
[the remark this answers is not in the document]
Comment (RosaWagner) [reply ofl/notosanssymbols2/METADATA.pb:7]: @simoncozens I downloaded the current live font and the font name is actually symbols2. I don't know why METADATA.pb was not consistent, but it is problematic to change the name in metadata.pb now…

cc @davelab6 @chrissimpkins

[the remark this answers is not in the document]
Comment (chrissimpkins) [reply ofl/notosanssymbols2/METADATA.pb:7]: I believe that this changes the API URL, correct?  And maybe the URL used to access the [f.g.c. page](https://fonts.google.com/noto/specimen/Noto+Sans+Symbols+2)?

[the remark this answers is not in the document]
Comment (simoncozens) [reply ofl/notosanssymbols2/METADATA.pb:7]: We can set a display_name but I don't know how much that helps.

[the remark this answers is not in the document]
Comment (RosaWagner) [reply ofl/notosanssymbols2/METADATA.pb:7]: For what happen in other PR previously; the display_name can't be used for that, and the font should be renamed so it matches the API name. But @davelab6 should maybe confirm.

File: ofl/notosanssymbols2/METADATA.pb (hunk 1 above)
Comment (vv-monsalve) [reply ofl/notosanssymbols2/METADATA.pb:25]: From our last Quarterly onboarding on-call meeting, I would say not. IIUIC they said these metadata fields were used for symbols fonts for which we don't have a centralized sample text.

Comment (vv-monsalve) [reply ofl/notosanssymbols2/METADATA.pb:25]: Similar situation as in [Bamum](https://github.com/google/fonts/pull/5412#event-7589621703)

File: ofl/notosanssymbols2/METADATA.pb (hunk 2 above)
[the remark this answers is not in the document]
Comment (RosaWagner) [reply ofl/notosanssymbols2/METADATA.pb:7]: @simoncozens we agreed on separating symbols from 2

[the remark this answers is not in the document]
Comment (RosaWagner) [reply ofl/notosanssymbols2/METADATA.pb:7]: and it should be updated in description/article so it is all consistent

[the remark this answers is not in the document]
Comment (simoncozens) [reply ofl/notosanssymbols2/METADATA.pb:7]: Ok, will fix on the Noto side.

**Comment by RosaWagner** [comment]
## Fontbakery report

Fontbakery version: 0.7.39.dev559+gf363acca

<details><summary><b>[1] Family checks</b></summary><div><details><summary>ℹ <b>INFO:</b> Check axis ordering on the STAT table.  (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/STAT/axis_order">com.google.fonts/check/STAT/axis_order</a>)</summary><div>


* ℹ **INFO** From a total of 1 font files, 1 of them (100.00%) lack a STAT table.

	And these are the most common STAT axis orderings:
	 [code: summary]
</div></details><br></div></details><details><summary><b>[20] NotoSansSymbols2-Regular.ttf</b></summary><div><details><summary>🔥 <b>FAIL:</b> Check Google Fonts glyph coverage. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage">com.google.fonts/check/glyph_coverage</a>)</summary><div>


* 🔥 **FAIL** Missing required codepoints:

	- 0x01CD (LATIN CAPITAL LETTER A WITH CARON)
 

	- And 0x01CE (LATIN SMALL LETTER A WITH CARON)
 [code: missing-codepoints]
</div></details><details><summary>🔥 <b>FAIL:</b> Check for METADATA subsets with zero support. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/unsupported_subsets">com.google.fonts/check/metadata/unsupported_subsets</a>)</summary><div>


* 🔥 **FAIL** Please remove the unrecognized subset 'symbols' from the METADATA.pb file. [code: unknown-subset]
</div></details><details><summary>🔥 <b>FAIL:</b> METADATA.pb: Designers are listed correctly on the Google Fonts catalog? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/designer_profiles">com.google.fonts/check/metadata/designer_profiles</a>)</summary><div>


* 🔥 **FAIL** The avatar filename provided seems to be incorrect: (https://raw.githubusercontent.com/google/fonts/master/catalog/designers/google/) [code: bad-avatar-filename]
</div></details><details><summary>🔥 <b>FAIL:</b> Do we have the latest version of FontBakery installed? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/fontbakery_version">com.google.fonts/check/fontbakery_version</a>)</summary><div>


* 🔥 **FAIL** Current Font Bakery version is 0.7.39.dev559+gf363acca, while a newer 0.8.10 is already available. Please upgrade it with 'pip install -U fontbakery' [code: outdated-fontbakery]
</div></details><details><summary>🔥 <b>FAIL:</b> Ensure dotted circle glyph is present and can attach marks. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/dotted_circle">com.google.fonts/check/dotted_circle</a>)</summary><div>


* 🔥 **FAIL** The following glyphs could not be attached to the dotted circle glyph:

	- acutecomb

	- gravecomb

	- tildecomb

	- uni0302

	- uni0304

	- uni0306

	- uni0307

	- uni0308

	- uni030A

	- uni030B 

	- And 5 more.

Use -F or --full-lists to disable shortening of long lists. [code: unattached-dotted-circle-marks]
</div></details><details><summary>⚠ <b>WARN:</b> Ensure files are not too large. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/file_size">com.google.fonts/check/file_size</a>)</summary><div>


* ⚠ **WARN** Font file is 1.2Mb; ideally it should be less than 1.0Mb [code: large-font]
</div></details><details><summary>⚠ <b>WARN:</b> METADATA.pb: Fontfamily is listed on Google Fonts API? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/listed_on_gfonts">com.google.fonts/check/metadata/listed_on_gfonts</a>)</summary><div>


* ⚠ **WARN** Family not found via Google Fonts API. [code: not-found]
</div></details><details><summary>⚠ <b>WARN:</b> A static fonts directory with at least two fonts must accompany variable fonts (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/repo/vf_has_static_fonts">com.google.fonts/check/repo/vf_has_static_fonts</a>)</summary><div>


* ⚠ **WARN** Please consider adding a subdirectory called "static/" and including in it static font files. [code: missing]
</div></details><details><summary>⚠ <b>WARN:</b> Ensure fonts have ScriptLangTags declared on the 'meta' table. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/meta/script_lang_tags">com.google.fonts/check/meta/script_lang_tags</a>)</summary><div>


* ⚠ **WARN** This font file does not have a 'meta' table. [code: lacks-meta-table]
</div></details><details><summary>⚠ <b>WARN:</b> Check if each glyph has the recommended amount of contours. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/contour_count">com.google.fonts/check/contour_count</a>)</summary><div>


* ⚠ **WARN** This check inspects the glyph outlines and detects the total number of contours in each of them. The expected values are infered from the typical ammounts of contours observed in a large collection of reference font families. The divergences listed below may simply indicate a significantly different design on some of your glyphs. On the other hand, some of these may flag actual bugs in the font such as glyphs mapped to an incorrect codepoint. Please consider reviewing the design and codepoint assignment of these to make sure they are correct.

The following glyphs do not have the recommended number of contours:

	- Glyph name: aogonek	Contours detected: 3	Expected: 2

	- Glyph name: uogonek	Contours detected: 2	Expected: 1

	- Glyph name: uni2611	Contours detected: 3	Expected: 2

	- Glyph name: aogonek	Contours detected: 3	Expected: 2

	- Glyph name: uni2611	Contours detected: 3	Expected: 2 

	- And Glyph name: uogonek	Contours detected: 2	Expected: 1
 [code: contour-count]
</div></details><details><summary>⚠ <b>WARN:</b> Check mark characters are in GDEF mark glyph class. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/gdef.html#com.google.fonts/check/gdef_mark_chars">com.google.fonts/check/gdef_mark_chars</a>)</summary><div>


* ⚠ **WARN** The following mark characters could be in the GDEF mark glyph class:
	 u101FD (U+101FD), u102E0 (U+102E0), uni20E2 (U+20E2) and uni20E3 (U+20E3) [code: mark-chars]
</div></details><details><summary>⚠ <b>WARN:</b> Do outlines contain any jaggy segments? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_jaggy_segments">com.google.fonts/check/outline_jaggy_segments</a>)</summary><div>


* ⚠ **WARN** The following glyphs have jaggy segments:

	* u101D1 (U+101D1): L<<158.0,655.0>--<202.0,553.0>>/L<<202.0,553.0>--<167.0,658.0>> = 4.899092453787774

	* u101D1 (U+101D1): L<<196.0,669.0>--<231.0,563.0>>/L<<231.0,563.0>--<221.0,676.0>> = 13.21538973442621

	* u101D1 (U+101D1): L<<290.0,688.0>--<289.0,558.0>>/L<<289.0,558.0>--<300.0,680.0>> = 4.71135362944517

	* u101D1 (U+101D1): L<<408.0,660.0>--<382.0,547.0>>/L<<382.0,547.0>--<424.0,635.0>> = 12.55629033344111

	* u101D1 (U+101D1): L<<451.0,623.0>--<410.0,538.0>>/L<<410.0,538.0>--<466.0,619.0>> = 8.907880578202827

	* u101D1 (U+101D1): L<<490.0,602.0>--<441.0,532.0>>/L<<441.0,532.0>--<501.0,590.0>> = 10.97900173252049

	* u101D3 (U+101D3): B<<230.0,133.5>-<227.0,152.0>-<225.0,180.0>>/B<<225.0,180.0>-<225.0,153.0>-<211.0,120.0>> = 4.085616779974799

	* u101D4 (U+101D4): B<<127.0,331.0>-<127.0,364.0>-<133.0,388.0>>/B<<133.0,388.0>-<125.0,373.0>-<120.0,326.0>> = 14.036243467926484

	* u101D5 (U+101D5): B<<262.0,599.0>-<295.0,599.0>-<341.0,560.0>>/B<<341.0,560.0>-<317.0,588.0>-<287.0,604.5>> = 9.106557599367749

	* u101D9 (U+101D9): B<<140.0,253.0>-<140.0,110.0>-<146.0,-22.0>>/B<<146.0,-22.0>-<154.0,23.0>-<158.5,91.5>> = 12.683160190042093 

	* And 212 more.

Use -F or --full-lists to disable shortening of long lists. [code: found-jaggy-segments]
</div></details><details><summary>⚠ <b>WARN:</b> Do outlines contain any semi-vertical or semi-horizontal lines? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_semi_vertical">com.google.fonts/check/outline_semi_vertical</a>)</summary><div>


* ⚠ **WARN** The following glyphs have semi-vertical/semi-horizontal lines:

	* exclam (U+0021): L<<100.0,174.0>--<98.0,714.0>>

	* exclam (U+0021): L<<127.0,714.0>--<125.0,174.0>>

	* exclamdown (U+00A1): L<<122.0,354.0>--<124.0,-186.0>>

	* exclamdown (U+00A1): L<<96.0,-186.0>--<98.0,354.0>>

	* u101D1 (U+101D1): L<<290.0,688.0>--<289.0,558.0>>

	* u101DD (U+101DD): L<<50.0,22.0>--<54.0,655.0>>

	* u101DD (U+101DD): L<<80.0,285.0>--<78.0,29.0>>

	* u101DD (U+101DD): L<<83.0,657.0>--<81.0,394.0>>

	* u102F5 (U+102F5): L<<373.0,188.0>--<764.0,190.0>>

	* u102F5 (U+102F5): L<<765.0,100.0>--<201.0,97.0>> 

	* And 28 more.

Use -F or --full-lists to disable shortening of long lists. [code: found-semi-vertical]
</div></details><details><summary>ℹ <b>INFO:</b> Show hinting filesize impact. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/hinting_impact">com.google.fonts/check/hinting_impact</a>)</summary><div>


* ℹ **INFO** Hinting filesize impact:

 |               | /Users/rosalie/Desktop/Symbols2_QA/fonts/ofl/notosanssymbols2/NotoSansSymbols2-Regular.ttf          |
 |:------------- | ---------------:|
 | Dehinted Size | 673.8kb |
 | Hinted Size   | 1.2Mb   |
 | Increase      | 512.4kb      |
 | Change        | 76.0 %  |
 [code: size-impact]
</div></details><details><summary>ℹ <b>INFO:</b> EPAR table present in font? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/epar">com.google.fonts/check/epar</a>)</summary><div>


* ℹ **INFO** EPAR table not present in font. To learn more see https://github.com/googlefonts/fontbakery/issues/818 [code: lacks-EPAR]
</div></details><details><summary>ℹ <b>INFO:</b> Is the Grid-fitting and Scan-conversion Procedure ('gasp') table set to optimize rendering? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/gasp">com.google.fonts/check/gasp</a>)</summary><div>


* ℹ **INFO** These are the ppm ranges declared on the gasp table:

PPM <= 65535:
	flag = 0x0F
	- Use grid-fitting
	- Use grayscale rendering
	- Use gridfitting with ClearType symmetric smoothing
	- Use smoothing along multiple axes with ClearType®
 [code: ranges]
</div></details><details><summary>ℹ <b>INFO:</b> Check for font-v versioning. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fontv">com.google.fonts/check/fontv</a>)</summary><div>


* ℹ **INFO** Version string is: "Version 2.005; ttfautohint (v1.8.4.7-5d5b)"
The version string must ideally include a git commit hash and either a "dev" or a "release" suffix such as in the example below:
"Version 1.3; git-0d08353-release" [code: bad-format]
</div></details><details><summary>ℹ <b>INFO:</b> Check samples can be rendered. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/can_render_samples">com.google.fonts/check/metadata/can_render_samples</a>)</summary><div>


* ℹ **INFO** No sample_glyphs on METADATA.pb [code: no-samples]
</div></details><details><summary>ℹ <b>INFO:</b> Font contains all required tables? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/required_tables">com.google.fonts/check/required_tables</a>)</summary><div>


* ℹ **INFO** This font contains the following optional tables:

	- cvt 

	- fpgm

	- loca

	- prep

	- GPOS

	- GSUB

	- gasp

	- vhea 

	- And vmtx [code: optional-tables]
</div></details><details><summary>ℹ <b>INFO:</b> List all superfamily filepaths (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/superfamily/list">com.google.fonts/check/superfamily/list</a>)</summary><div>


* ℹ **INFO** /Users/rosalie/Desktop/Symbols2_QA/fonts/ofl/notosanssymbols2 [code: family-path]
</div></details><br></div></details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 5 | 8 | 58 | 8 | 154 | 0 |
| 0% | 2% | 3% | 25% | 3% | 66% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **PASS**
* **DEBUG**

**Comment by RosaWagner** [comment]
## Fontbakery report

Fontbakery version: 0.8.10

<details><summary><b>[1] Family checks</b></summary><div><details><summary>ℹ <b>INFO:</b> Check axis ordering on the STAT table.  (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/STAT/axis_order">com.google.fonts/check/STAT/axis_order</a>)</summary><div>


* ℹ **INFO** From a total of 1 font files, 1 of them (100.00%) lack a STAT table.

	And these are the most common STAT axis orderings:
	 [code: summary]
</div></details><br></div></details><details><summary><b>[19] NotoSansSymbols2-Regular.ttf</b></summary><div><details><summary>🔥 <b>FAIL:</b> Check Google Fonts glyph coverage. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/glyph_coverage">com.google.fonts/check/glyph_coverage</a>)</summary><div>


* 🔥 **FAIL** Missing required codepoints:

	- 0x01CD (LATIN CAPITAL LETTER A WITH CARON)
 

	- And 0x01CE (LATIN SMALL LETTER A WITH CARON)
 [code: missing-codepoints]
</div></details><details><summary>🔥 <b>FAIL:</b> Check for METADATA subsets with zero support. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/unsupported_subsets">com.google.fonts/check/metadata/unsupported_subsets</a>)</summary><div>


* 🔥 **FAIL** Please remove the unrecognized subset 'symbols' from the METADATA.pb file. [code: unknown-subset]
</div></details><details><summary>🔥 <b>FAIL:</b> METADATA.pb: Designers are listed correctly on the Google Fonts catalog? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/designer_profiles">com.google.fonts/check/metadata/designer_profiles</a>)</summary><div>


* 🔥 **FAIL** The avatar filename provided seems to be incorrect: (https://raw.githubusercontent.com/google/fonts/master/catalog/designers/google/) [code: bad-avatar-filename]
</div></details><details><summary>🔥 <b>FAIL:</b> Ensure dotted circle glyph is present and can attach marks. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/dotted_circle">com.google.fonts/check/dotted_circle</a>)</summary><div>


* 🔥 **FAIL** The following glyphs could not be attached to the dotted circle glyph:

	- acutecomb

	- gravecomb

	- tildecomb

	- uni0302

	- uni0304

	- uni0306

	- uni0307

	- uni0308

	- uni030A

	- uni030B 

	- And 5 more.

Use -F or --full-lists to disable shortening of long lists. [code: unattached-dotted-circle-marks]
</div></details><details><summary>⚠ <b>WARN:</b> Ensure files are not too large. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/file_size">com.google.fonts/check/file_size</a>)</summary><div>


* ⚠ **WARN** Font file is 1.2Mb; ideally it should be less than 1.0Mb [code: large-font]
</div></details><details><summary>⚠ <b>WARN:</b> METADATA.pb: Fontfamily is listed on Google Fonts API? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/listed_on_gfonts">com.google.fonts/check/metadata/listed_on_gfonts</a>)</summary><div>


* ⚠ **WARN** Family not found via Google Fonts API. [code: not-found]
</div></details><details><summary>⚠ <b>WARN:</b> A static fonts directory with at least two fonts must accompany variable fonts (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/repo/vf_has_static_fonts">com.google.fonts/check/repo/vf_has_static_fonts</a>)</summary><div>


* ⚠ **WARN** Please consider adding a subdirectory called "static/" and including in it static font files. [code: missing]
</div></details><details><summary>⚠ <b>WARN:</b> Ensure fonts have ScriptLangTags declared on the 'meta' table. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/meta/script_lang_tags">com.google.fonts/check/meta/script_lang_tags</a>)</summary><div>


* ⚠ **WARN** This font file does not have a 'meta' table. [code: lacks-meta-table]
</div></details><details><summary>⚠ <b>WARN:</b> Check if each glyph has the recommended amount of contours. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/contour_count">com.google.fonts/check/contour_count</a>)</summary><div>


* ⚠ **WARN** This check inspects the glyph outlines and detects the total number of contours in each of them. The expected values are infered from the typical ammounts of contours observed in a large collection of reference font families. The divergences listed below may simply indicate a significantly different design on some of your glyphs. On the other hand, some of these may flag actual bugs in the font such as glyphs mapped to an incorrect codepoint. Please consider reviewing the design and codepoint assignment of these to make sure they are correct.

The following glyphs do not have the recommended number of contours:

	- Glyph name: aogonek	Contours detected: 3	Expected: 2

	- Glyph name: uogonek	Contours detected: 2	Expected: 1

	- Glyph name: uni2611	Contours detected: 3	Expected: 2

	- Glyph name: aogonek	Contours detected: 3	Expected: 2

	- Glyph name: uni2611	Contours detected: 3	Expected: 2 

	- And Glyph name: uogonek	Contours detected: 2	Expected: 1
 [code: contour-count]
</div></details><details><summary>⚠ <b>WARN:</b> Check mark characters are in GDEF mark glyph class. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/gdef.html#com.google.fonts/check/gdef_mark_chars">com.google.fonts/check/gdef_mark_chars</a>)</summary><div>


* ⚠ **WARN** The following mark characters could be in the GDEF mark glyph class:
	 u101FD (U+101FD), u102E0 (U+102E0), uni20E2 (U+20E2) and uni20E3 (U+20E3) [code: mark-chars]
</div></details><details><summary>⚠ <b>WARN:</b> Do outlines contain any jaggy segments? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_jaggy_segments">com.google.fonts/check/outline_jaggy_segments</a>)</summary><div>


* ⚠ **WARN** The following glyphs have jaggy segments:

	* u101D1 (U+101D1): L<<158.0,655.0>--<202.0,553.0>>/L<<202.0,553.0>--<167.0,658.0>> = 4.899092453787774

	* u101D1 (U+101D1): L<<196.0,669.0>--<231.0,563.0>>/L<<231.0,563.0>--<221.0,676.0>> = 13.21538973442621

	* u101D1 (U+101D1): L<<290.0,688.0>--<289.0,558.0>>/L<<289.0,558.0>--<300.0,680.0>> = 4.71135362944517

	* u101D1 (U+101D1): L<<408.0,660.0>--<382.0,547.0>>/L<<382.0,547.0>--<424.0,635.0>> = 12.55629033344111

	* u101D1 (U+101D1): L<<451.0,623.0>--<410.0,538.0>>/L<<410.0,538.0>--<466.0,619.0>> = 8.907880578202827

	* u101D1 (U+101D1): L<<490.0,602.0>--<441.0,532.0>>/L<<441.0,532.0>--<501.0,590.0>> = 10.97900173252049

	* u101D3 (U+101D3): B<<230.0,133.5>-<227.0,152.0>-<225.0,180.0>>/B<<225.0,180.0>-<225.0,153.0>-<211.0,120.0>> = 4.085616779974799

	* u101D4 (U+101D4): B<<127.0,331.0>-<127.0,364.0>-<133.0,388.0>>/B<<133.0,388.0>-<125.0,373.0>-<120.0,326.0>> = 14.036243467926484

	* u101D5 (U+101D5): B<<262.0,599.0>-<295.0,599.0>-<341.0,560.0>>/B<<341.0,560.0>-<317.0,588.0>-<287.0,604.5>> = 9.106557599367749

	* u101D9 (U+101D9): B<<140.0,253.0>-<140.0,110.0>-<146.0,-22.0>>/B<<146.0,-22.0>-<154.0,23.0>-<158.5,91.5>> = 12.683160190042093 

	* And 212 more.

Use -F or --full-lists to disable shortening of long lists. [code: found-jaggy-segments]
</div></details><details><summary>⚠ <b>WARN:</b> Do outlines contain any semi-vertical or semi-horizontal lines? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/<Section: Outline Correctness Checks>.html#com.google.fonts/check/outline_semi_vertical">com.google.fonts/check/outline_semi_vertical</a>)</summary><div>


* ⚠ **WARN** The following glyphs have semi-vertical/semi-horizontal lines:

	* exclam (U+0021): L<<100.0,174.0>--<98.0,714.0>>

	* exclam (U+0021): L<<127.0,714.0>--<125.0,174.0>>

	* exclamdown (U+00A1): L<<122.0,354.0>--<124.0,-186.0>>

	* exclamdown (U+00A1): L<<96.0,-186.0>--<98.0,354.0>>

	* u101D1 (U+101D1): L<<290.0,688.0>--<289.0,558.0>>

	* u101DD (U+101DD): L<<50.0,22.0>--<54.0,655.0>>

	* u101DD (U+101DD): L<<80.0,285.0>--<78.0,29.0>>

	* u101DD (U+101DD): L<<83.0,657.0>--<81.0,394.0>>

	* u102F5 (U+102F5): L<<373.0,188.0>--<764.0,190.0>>

	* u102F5 (U+102F5): L<<765.0,100.0>--<201.0,97.0>> 

	* And 28 more.

Use -F or --full-lists to disable shortening of long lists. [code: found-semi-vertical]
</div></details><details><summary>ℹ <b>INFO:</b> Show hinting filesize impact. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/hinting_impact">com.google.fonts/check/hinting_impact</a>)</summary><div>


* ℹ **INFO** Hinting filesize impact:

 |               | /Users/rosalie/Desktop/Symbols2_QA/fonts/ofl/notosanssymbols2/NotoSansSymbols2-Regular.ttf          |
 |:------------- | ---------------:|
 | Dehinted Size | 673.8kb |
 | Hinted Size   | 1.2Mb   |
 | Increase      | 512.4kb      |
 | Change        | 76.0 %  |
 [code: size-impact]
</div></details><details><summary>ℹ <b>INFO:</b> EPAR table present in font? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/epar">com.google.fonts/check/epar</a>)</summary><div>


* ℹ **INFO** EPAR table not present in font. To learn more see https://github.com/googlefonts/fontbakery/issues/818 [code: lacks-EPAR]
</div></details><details><summary>ℹ <b>INFO:</b> Is the Grid-fitting and Scan-conversion Procedure ('gasp') table set to optimize rendering? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/gasp">com.google.fonts/check/gasp</a>)</summary><div>


* ℹ **INFO** These are the ppm ranges declared on the gasp table:

PPM <= 65535:
	flag = 0x0F
	- Use grid-fitting
	- Use grayscale rendering
	- Use gridfitting with ClearType symmetric smoothing
	- Use smoothing along multiple axes with ClearType®
 [code: ranges]
</div></details><details><summary>ℹ <b>INFO:</b> Check for font-v versioning. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/fontv">com.google.fonts/check/fontv</a>)</summary><div>


* ℹ **INFO** Version string is: "Version 2.005; ttfautohint (v1.8.4.7-5d5b)"
The version string must ideally include a git commit hash and either a "dev" or a "release" suffix such as in the example below:
"Version 1.3; git-0d08353-release" [code: bad-format]
</div></details><details><summary>ℹ <b>INFO:</b> Check samples can be rendered. (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/can_render_samples">com.google.fonts/check/metadata/can_render_samples</a>)</summary><div>


* ℹ **INFO** No sample_glyphs on METADATA.pb [code: no-samples]
</div></details><details><summary>ℹ <b>INFO:</b> Font contains all required tables? (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/required_tables">com.google.fonts/check/required_tables</a>)</summary><div>


* ℹ **INFO** This font contains the following optional tables:

	- cvt 

	- fpgm

	- loca

	- prep

	- GPOS

	- GSUB

	- gasp

	- vhea 

	- And vmtx [code: optional-tables]
</div></details><details><summary>ℹ <b>INFO:</b> List all superfamily filepaths (<a href="https://font-bakery.readthedocs.io/en/stable/fontbakery/profiles/universal.html#com.google.fonts/check/superfamily/list">com.google.fonts/check/superfamily/list</a>)</summary><div>


* ℹ **INFO** /Users/rosalie/Desktop/Symbols2_QA/fonts/ofl/notosanssymbols2 [code: family-path]
</div></details><br></div></details>
### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 0 | 4 | 8 | 55 | 8 | 152 | 0 |
| 0% | 2% | 4% | 24% | 4% | 67% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **PASS**
* **DEBUG**

**Pull request merged**